packages/epo-react-lib/src/atomic/Video/Video.tsx

```diff
@@ -1,15 +1,11 @@
 import { FunctionComponent } from "react";
-import ReactPlayer from "react-player/youtube";
+import ReactPlayer, { YouTubePlayerProps } from "react-player/youtube";
 import * as Styled from "./styles";
 
-export interface VideoProps {
-  url: string;
-}
-
-const Video: FunctionComponent<VideoProps> = ({ url }) => {
+const Video: FunctionComponent<YouTubePlayerProps> = (props) => {
```

Comment:
instead of reimplementing a subset of the props in our interfaces, let's just use the entire props definition out of react-player.

Reply:
Good thinking.